doc/source/notebooks/02_node_populations.ipynb

```diff
@@ -54,7 +56,7 @@
     }
    ],
    "source": [
-    "circuit.nodes"
+    "circuit.nodes.population_names"
    ]
   },
   {
```

Comment:
L66: 
```
In this example we can see there is only one node population, which is named `All`. We can access the node population using the name as the key:
```
Should be something like:
```
In this example, we can see there are several node populations. We can access a node population using its name as the key:
```

Reply:
Thanks for spotting this, I've updated the text.